src/Util/ModuleUtils.php

```diff
@@ -74,18 +103,53 @@ public function downloadMainClass(string $moduleName, Version $version): void
 
         $response = $this->client->get(sprintf(self::GITHUB_MAIN_CLASS_ENDPOINT, $moduleName, $version->getTag(), $moduleName));
         file_put_contents($path . '/' . $moduleName . '.php', $response->getBody());
-        $this->saveDownloadUrl($moduleName, $version);
     }
 
-    /**
-     * @return Module[]
-     */
-    public function getLocalModules(): array
+    public function download(string $moduleName, Version $version): void
+    {
+        $path = join('/', [$this->moduleDir, $moduleName, $version->getTag()]);
+        if (!is_dir($path)) {
+            mkdir($path, 0777, true);
```

Comment:
0777 😄 ?

Reply:
mmh, yes? What's the issue with this? 😅

Reply:
0755? 0644? But 0777 is usually NOT the good option